proteomic_aggregation/python/proteomic_aggregation.ipynb

```diff
@@ -7,8 +7,7 @@
     "# Proteomic Data Aggregation and Visualization\n",
```

Comment:
<p>We may want to clean up this cell output from the released notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>